custom/intrahealth/sqldata.py

```diff
@@ -92,6 +92,8 @@ def filters(self):
         #Filtering is done directly in columns method(CountUniqueColumn).
         filters = super(ConventureData, self).filters
         filters.append(AND([GTE('real_date_repeat', "strsd"), LTE('real_date_repeat', "stred")]))
+        if 'archived_locations' in self.config:
+            filters.append(NOT(IN('location_id', 'archived_locations')))
```

Comment:
(not actionable) it would be nice if we could tag the data directly to avoid passing the list of archived locations around and possibly doing a big `NOT IN` query, but that seems like it would be a lot of work to keep in sync.